monkey/monkey_island/cc/repository/mongo_credentials_repository.py

```diff
@@ -0,0 +1,98 @@
+from typing import Sequence
+
+from flask_pymongo import PyMongo
```

Comment:
Use [pymongo](https://pymongo.readthedocs.io/en/stable/) instead of flask_pymongo.